pages/services/edge-lb/1.2/pool-configuration/index.md

```diff
@@ -12,18 +12,20 @@ This is a reference for all possible Edge-LB pool configuration values, as well
 
 # Configuration Reference
 
-The configuration reference containing all possible options and descriptions of each can be found by running the following command:
+The API reference, in swagger format, containing all possible options and short descriptions of each endpoint can be found by running the following command:
```

Comment:
I think this period is very long. What about?

`The API reference, in swaggger format, can be found by running the command bellow. It contains all possible options and a short description for each endpoint`